tools/simulate_geometry.py

```
@@ -0,0 +1,113 @@
+            snapshot = demo_snapshots[0] if demo_snapshots else {}
+            
+            t = time.time()
+            
+            manifold_payload = {
+                "type": "manifold_update",
+                "timestamp": t,
+                "parameters": {
+                    "uMin": 0, "uMax": 1,
+                    "vMin": 0, "vMax": 1,
+                    "t": t % 100,
+                    "k": -0.5 + (0.1 * random.random()), # Hyperbolic baseline
```

Ruff (0.14.10)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)